server/bundler/assets-writer.js

```diff
@@ -85,7 +85,7 @@ Object.assign( AssetsWriter.prototype, {
 				} )
 			);
 
-			self.outputStream.write( JSON.stringify( statsToOutput, null, '\t' ) );
+			self.outputStream.end( JSON.stringify( statsToOutput, null, '\t' ), () => callback() );
```

Comment:
Can we simplify `() => callback()` to just `callback`?

Reply:
Yes, the callback of `.end` doesn't seem to get any arguments so that should be possible.